api/scripts/certification/get-user-certifications-eligibility.js

```diff
@@ -10,23 +10,41 @@ import { usecases } from '../../lib/domain/usecases/index.js';
 import { learningContentCache as cache } from '../../lib/infrastructure/caches/learning-content-cache.js';
 import * as placementProfileService from '../../lib/domain/services/placement-profile-service.js';
 import * as certificationBadgesService from '../../lib/domain/services/certification-badges-service.js';
-// Usage: node scripts/get-certifications-eligibility 1234
-
 import { disconnect } from '../../db/knex-database-connection.js';
 import { temporaryStorage } from '../../lib/infrastructure/temporary-storage/index.js';
 
-async function getUserCertificationsEligibility(userId) {
+/**
+ * DESCRIPTION
+ *    Will display certifications eligibiliby for a user at a specific date
+ *    Note: by default date is full today's date at 23h 59m 59s
+ *
+ * USAGE
+ *    $ node get-user-certifications-eligibility.js "<userId>" "[<YYYY-MM-DD>]" "[HH:mm:ss]"
+ *
+ * EXAMPLES:
+ *    # Today at 23h59:59
+ *    $ node get-user-certifications-eligibility.js 147114
+ *
+ *    # On 26/07/2023 at 23h 59m 59s
+ *    $ node get-user-certifications-eligibility.js 147114 2023-07-26
+ *
+ *    # On 22/01/2000 at 11h 52m 00s
+ *    $ node get-user-certifications-eligibility.js 147114 2000-01-22 11:52
+ */
+
+async function getUserCertificationsEligibility({ userId, limitDate }) {
   logger.info('Starting script get-user-certifications-eligibility');
 
   const { pixCertificationEligible, eligibleComplementaryCertifications } =
     await usecases.getUserCertificationEligibility({
       userId,
       placementProfileService,
       certificationBadgesService,
+      limitDate,
     });
 
   const complementaires = eligibleComplementaryCertifications.map(({ label }) => label).join(', ') || '❌';
-  console.log(`Eligibilité user ${userId}`);
+  console.log(`Eligibilité utilisateur ${userId} à ${limitDate.toISOString()}`);
   console.log(`PIX: ${pixCertificationEligible ? '✅' : '❌'}`);
   console.log(`COMPLEMENTAIRES: ${complementaires}`);
```

Comment:
On ne peut en avoir qu'une désormais :D 

```suggestion
  console.log(`COMPLEMENTAIRE: ${complementaires}`);
```

Reply:
@aceol tu te souviens pourquoi on avait pas refacto ici ?

Reply:
Done (je ne touche pas au map par contre)

Reply:
Rollback, on peut être éligible à plusieurs